File: anul 2/poo/lab2/lab2_2.cpp
#include<iostream>

using namespace std;

class Car{
public:
string manufacture;
int year;
int engineSize;
Car(string m,int y,int e);
Car(int e,string manufacture);
Car(string m,int y);

};
Car::Car(string m,int y,int e)
{
    manufacture=m;
    year=y;
    engineSize=e;
}
Car::Car(int e, string manufacture){};

int main()
{

}

File: anul 2/semestrul 1/poo/kotlin/src/lab2/lab2_2.kt
class Car(val brand: String, val model: String) {
    constructor(brand: String, model: String, year: Int) : this(brand, model) {
        println("Secondary constructor called")
        println("Year: $year")
    }
}

fun main() {
    val car1 = Car("Toyota", "Camry")
    val car2 = Car("Honda", "Civic", 2022)
}
